tests/BlazingSQLTest/Runner/runTest.py

```diff
@@ -1676,28 +1148,30 @@ def run_query(
                         load_time,
                         engine_time,
                         total_time,
-                        comparing
-                    )
-            else:
-                print_query_results2(
-                    query, queryId, queryType, result_gdf.error_message
-                )
-
-def run_query_log(
-    bc,
-    query,
-    queryId,
-    queryType,
-    **kwargs
-):
-    result_gdf = None
-    error_message = ""
-    message_validation = ""
-
-    try:
-        result_gdf = bc.log(query)
-    except Exception as e:
-        error_message=str(e)
+                        comparing,
+                        compareResults,
+                        resultFile)
+      
+def results_processing(result_gdf, 
```

Comment:
Could you please add comments to document this function. What does it do? what are all these parameters?